gympp/include/gympp/Robot.h

```diff
@@ -63,6 +65,16 @@ struct gympp::Limit
     double max = std::numeric_limits<double>::max();
 };
 
+struct gympp::ContactData
+{
+    std::string bodyA;
+    std::string bodyB;
+    std::array<double, 3> depth;
+    std::array<double, 6> wrench;
+    std::array<double, 3> normal;
+    std::array<double, 3> position;
```

Comment:
In which frame of reference is this position expressed? The world frame?

Reply:
It follows the same convention used by gazebo. I'm not sure if there's a documentation somewhere, though contact position is currently expressed in world coordinates.